plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: palo_alto_cortex_xdr
 title: Palo Alto Cortex XDR
 description: Stop modern attacks with the industry's first extended detection and response platform that spans your endpoints, network and cloud data
-version: 2.0.0
+version: 3.0.0
```

Comment:
Change to 2.1.0 because there are no backwards incompatible changes

Reply:
Done, thanks!